### ISSUE TITLE
FileNotFoundException in WPF project with ReactiveUI.WPF installed

### ISSUE THREAD

**Issue by Janek91** [reactiveui/ReactiveUI#1686]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

After installation ReactiveUI.WPF nuget besides ReactiveUI, both in 8.3.1 version - application is not able to start due to missing file exception. Missing file is System.Reactive.Core in 3.0.1000.0 version, which is referenced by System.Reactive.Windows.Threading in 3.0.1000.0 version, which is referenced by ReactiveUI.WPF in version 8.3.1. Though, ReactiveUI in version 8.3.1 has System.Reactive.Core referenced in 3.0.3000.0 version and this version is winning.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**

Install ReactiveUI.WPF nuget package as PackageReference in WPF application project and try to run it.

**What is the expected behavior?**

To be able to run WPF application when ReactiveUI.WPF nuget is installed.

**What is the motivation / use case for changing the behavior?**

**Which versions of ReactiveUI, and which platform / OS are affected by this issue? Did this work in previous versions of ReativeUI? Please also test with the [latest stable](https://www.nuget.org/profiles/reactiveui) and [development snapshot](https://reactiveui.net/docs/getting-started/installation/nuget-packages/development-package-feed)**

ReactiveUI.WPF 8.0.0-8.3.1 - .NET 4.6.2

**Other information (e.g. stacktraces, related issues, suggestions how to fix)**

**Comment by glennawatson** [comment]
Most likely you've had System.Reactive 4.0.0 installed. 

They have a requirement now if you have binaries compiled against System.Reactive < 4 that you add a extra nuget package https://www.nuget.org/packages/System.Reactive.Compatibility/

Try adding that to see if it resolves your issues.

**Comment by chuuddo** [comment]
Don't have FileNotFoundException, but msbuild prints warning about "AutoGenerateBindingRedirects" for System.Reactive.Core from version 3.0.1000.0 to version 3.0.3000.0.

```xml
    <PackageReference Include="ReactiveUI.Events.WPF" Version="8.3.1" />
    <PackageReference Include="ReactiveUI.WPF" Version="8.3.1" />
```

**Comment by Janek91** [comment]
I got project with only ReactiveUI.WPF in 8.3.1 version installed via PackageReference. Dependencies are automatically resolved during build.

**Comment by glennawatson** [comment]
Turn on Fusion Bind logging and see what's causing the issue maybe? I suspect a older reference.  https://www.hanselman.com/blog/BackToBasicsUsingFusionLogViewerToDebugObscureLoaderErrors.aspx

If you turn Fusion logging on also, it'll also usually populate a field in the Exception which you can observe also. 

Let us know how that goes. I am able to use RxUI with 8.3.1 without binding issues.

**Comment by BenKenoby** [comment]
Changing the target framework to 4.7.2 removed the error for me.

**Comment by worldbeater** [comment]
Another solution is to install `ReactiveUI` package instead of `ReactiveUI.WPF`, and paste this line of code into you composition root (App.xaml.cs constructor, for example):
```csharp
RxApp.MainThreadScheduler = new WaitForDispatcherScheduler(() => DispatcherScheduler.Current);
```

**Comment by glennawatson** [comment]
Other stuff in the wpf project that just that registration. You loose the ReactiveUserControl etc.

**Comment by glennawatson** [comment]
i am pretty confident this will be fixed in 8.4.4 and current myget.
